README-cdk.md

```diff
@@ -135,8 +135,8 @@ Start two CDK/minishift clusters with OCP 3.11 called `cluster1` and
 walkthrough, so it's recommended that you adhere to them:
 
 ~~~sh
-    cdk start --profile cluster1 --ocp-tag v3.11.16
-    cdk start --profile cluster2 --ocp-tag v3.11.16
+cdk start --profile cluster1 --openshift-version v3.11.16
+cdk start --profile cluster2 --openshift-version v3.11.16
```

Comment:
Good to know about the change to `--openshift-version` thanks.

Shouln't we update v3.11.16 to the latest too? (v3.11.51)

Reply:
Agreed. Let me test with 3.11.51 and I'll update the PR.